libpod/runtime.go

```diff
@@ -701,6 +699,8 @@ var libimageEventsMap = map[libimage.EventType]events.Status{
 // events on the libimage.Runtime.  The gourtine will be cleaned up implicitly
 // when the main() exists.
 func (r *Runtime) libimageEvents() {
+	r.libimageEventsShutdown = make(chan bool)
```

Comment:
Wouldn't be best do this at very creation of runtime like `newRuntimeFromConfig()` and maybe some tests? Then we might omit the nil checks because it's always initialized.